packages/pyright-internal/src/common/configOptions.ts

```diff
@@ -1469,6 +1469,8 @@ export class ConfigOptions {
     // Overrides the default timeout for file enumeration operations.
     fileEnumerationTimeoutInSec?: number;
 
+    overloadConsistencyStrictness: 'allow_wide' | 'strict' = 'strict';
```

Comment:
for consistency with other similar settings i think this is a better name
```suggestion
    structOverloadConsistency: 'allow_wide' | 'strict' = 'strict';
```

Reply:
k, do you think the values are k?

Reply:
Oh i totqlly missed that, i think it should just be bool like the others, unless you plan to add more in the future?